replication.proto

```diff
@@ -43,6 +43,14 @@ message EnableVolumeReplicationRequest {
   // This field SHALL be used by the CO in subsequent calls to refer to
   // this volume.
   string volume_id = 1;
+  // The identifier for the target volume, generated by the plugin during
```

Comment:
How this is different from the volume_id?

Reply:
the purpose of this change is to add support for storage systems that don't have a single external volume id for both underlying volumes.
the target volume id is an existing volume, that doesn't have any (valuable) data, with which the user syncs the data of the volume of `volume_id`.
hope it's clear

Reply:
note that since it's optional, the spec will continue to support a single `volume_id` as well, in the same way it used to